### DISCUSSION TITLE
How to get the Rich traceback has a function like `traceback.format_exc()`

### DISCUSSION THREAD

**Discussion by shaoerkuai** [Textualize/rich#3434] [Q&A] [answered]
When I'm using Robot Framework to produce a RPA ,and want to use Rich to print the traceback:
Robot requires users use `import robot.api.logger` robot logger to print logs, if just using `print_exception()` it will not be recognized as a log.
Generally, I formly use `traceback.format_exec()` to get the string to be print and print it via Robot logger.

**Comment by willmcgugan** [comment, marked as answer]
If you use RichHandler (search the docs), it will log tracebacks.

Or you can [print tracebacks](https://rich.readthedocs.io/en/latest/traceback.html)
